server/resources/Project.py

```diff
@@ -0,0 +1,45 @@
+from flask_restful import Resource
+from flask import request
+from marshmallow import ValidationError
+from models import db, ma, ProjectModel, ProjectSchema
+
+PROJECT_ALREADY_EXISTS = "A project with that name already exists."
+CREATED_SUCCESSFULLY = "Project created successfully."
+PROJECT_NOT_FOUND = "Project not found."
+PROJECT_DELETED = "Project deleted."
+
+project_schema = ProjectSchema()
+
+
+class ProjectCreate(Resource):
```

Comment:
I think this should be in the class below instead of having another class for create